user_sessions/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class UserSessionsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'user_sessions'
+    verbose_name = 'User Sessions'
```

Comment:
```suggestion
    verbose_name = 'User Session'
```

You should use singular here. Django will add an `s` to any places where it needs to be pluralizes. See also `verbose_name_plural`.